src/adapter/http/src/middleware/authentication_layer.rs

```diff
@@ -68,6 +70,9 @@ impl<Svc> AuthenticationMiddleware<Svc> {
                             .await
                     },
                 )
+                .instrument(tracing::debug_span!(
+                    "AuthenticationMiddleware::current_account_subject"
```

Comment:
I only which there was some indication in the name that this span represents a DB transaction, otherwise we'll have to remember ourselves which parts of the code open them. Can't think of a good name though...

Reply:
We will still have "transactional" span inside.